### PR TITLE
feat: add weserv provider

### PR DESCRIPTION
**Pull request by nandi95**
Most options are in the keymap, the documentation is just a link to the website. That could be improved but frankly that's a lot of work. Maybe someone can put ai to use and summaries the doc for the provider documentation

### CODE REVIEW

**Comment by nandi95** [comment]
not sure how to add the documentation in either

File: docs/content/3.providers/weserv.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+title: Weserv
+description: Optimize images with Weserv's dynamic image transformation service.
+---
+
+Weserv offers a wide range of image transformation for all JPEG, PNG, and GIF files.
+
+
+## Modifiers
+
+In addition to `height` and `width`, the Weserv provider numerous modifiers documentation [here](https://images.weserv.nl/docs/quick-reference.html/).
```
Comment (Atinux) [inline docs/content/3.providers/weserv.md:11]:
```suggestion
In addition to `height` and `width`, the Weserv provider numerous modifiers documentation [here](https://images.weserv.nl/docs/quick-reference.html).
```

**Comment by Atinux** [comment]
Are we not talking about https://wsrv.nl/ directly instead of WeServ?

**Comment by Atinux** [comment]
Would love to have feedback from @kleisauke on this 🙏

**Comment by nandi95** [comment]
> Are we not talking about https://wsrv.nl/ directly instead of WeServ?

One and the same https://github.com/weserv/images

**Comment by nandi95** [comment]
I'm confused, this somehow has been wiped? Should I remake the PR? @danielroe

**Comment by danielroe** [comment]
Yes, please do remake the PR. The issue is that the GitHub actions bot responsible for keeping your fork's main branch up-to-date with upstream overwrote your changes, and the PR was autoclosed.

**Pull request merged**